app/precompiled.py

```diff
@@ -110,17 +110,23 @@ def rect(self, pt1, pt2):
 @statsd(namespace='template_preview')
 def sanitise_precompiled_letter():
     encoded_string = request.get_data()
+    allow_international_letters = (
+        request.args.get('allow_international_letters') == 'true'
```

Comment:
This is a `POST` endpoint only, so you won't be able to put `allow_international_letters` in `request.args`.

Also, given that api / admin will probably be passing through a boolean that has been converted to a string I guess we'd want to compare the value to `'True'` not `'true'` perhaps

Reply:
You can still `post` to a URL with with query string parameters. The alternative was to have two URLs, eg `/sanitise` and `/sanitise-international` but this felt less RESTful and would make the deployment order more rigid. We can’t put anything in the body of the request because it’s just the string encoded PDF, not JSON.

Putting capitalisation in URLs feels a bit wrong to me. I’d happy to make sure that the API/admin pass through the right string, rather than doing any type casting.

Reply:
👍 I've not seen request args and POST mixed before, but I can see how it makes sense here